boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java

```diff
@@ -137,4 +141,55 @@ private void writeSpringBootDockerFile(File dockerFile, String libertyImage) thr
                 + "/config/dropins/spring/thin-${APP_FILE}");
         Files.write(dockerFile.toPath(), lines, Charset.forName("UTF-8"));
     }
+
+    public void createDockerIgnore() throws Exception {
```

Comment:
maybe should be called `writeDockerIgnore()` or `mergeDockerIgnore()` right at the top-level here (instead of further down)?